src/admin-views/widgets/list.php

```diff
@@ -23,6 +23,39 @@
 }
 
 foreach ( $admin_fields as $field_id => $field ) {
+	// Handle a section start. May contain a section title.
+	if ( false !== stripos( $field_id, 'section_start' ) ) {
+		$section_classes = [ 'tribe-events-widget-admin-form__input-section' ];
+		if ( ! empty( $field['classes'] ) ) {
+			$classes = $field['classes'];
+
+			if ( ! is_array( $classes ) ) {
+				$classes = explode( ' ', $classes );
+			}
```

Comment:
I would suggest picking one, whether it's an array or string, and sticking with it. This sort of logic shouldn't be done here. Also add `$classes` to the docblock above.